test/parallel/test-fs-read-stream-throw-type-error.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
```

Comment:
These tests should probably check for `null`. It might be a good idea to also verify that string, object, and `undefined` do not throw.